scripts/gulpfiles/build_tasks.js

```diff
@@ -409,30 +411,58 @@ goog.require('Blockly.requires');
 };
 
 /**
- * This task builds Blockly's lang files.
- *     msg/*.js
+ * This task regenrates msg/json/en.js and msg/json/qqq.js from
+ * msg/messages.js.
  */
-function buildLangfiles(done) {
+function generateLangfiles(done) {
   // Run js_to_json.py
-  const jsToJsonCmd = `python ./scripts/i18n/js_to_json.py \
---input_file ${path.join('msg', 'messages.js')} \
---output_dir ${path.join('msg', 'json')} \
---quiet`;
+  const jsToJsonCmd = `python scripts/i18n/js_to_json.py \
+      --input_file ${path.join('msg', 'messages.js')} \
+      --output_dir ${path.join('msg', 'json')} \
+      --quiet`;
   execSync(jsToJsonCmd, { stdio: 'inherit' });
 
-  // Run create_messages.py
+  console.log(`
+Regenerated several flies in msg/json/.  Now run
+
+    git diff msg/json/*.json
+
+and check that operation has not overwritten any modifications made to
+hints, etc. by the TranslateWiki volunteers.  If it has, backport
+their changes to msg/messages.js and re-run 'npm run generate:langfiles'.
+
+Once you are satisfied that any new hints have been backported you may
+go ahead and commit the changes, but note that the generate script
+will have removed the translator credits - be careful not to commit
+this removal!
+`);
+
+  done();
+};
+
+/**
+ * This task builds Blockly's lang files.
+ *     msg/*.js
+ */
+function buildLangfiles(done) {
+  // Create output directory.
+  // BUG(cpcallen): this probably doesn't work on Windows.
```

Comment:
When checking in, use TODO(#issuenumber)

Reply:
Done.